app/assets/javascripts/discourse/app/lib/ajax-error.js

```diff
@@ -1,7 +1,8 @@
 import I18n from "I18n";
 import { getOwner } from "discourse-common/lib/get-owner";
+import { htmlSafe } from "@ember/template";
 
-export function extractError(error, defaultMessage) {
+function extractErrorInfo(error, defaultMessage) {
```

Comment:
```suggestion
function extractErrorInfo(error, defaultMessage = undefined) {
```

This would mean you could simply call `extractErrorInfo(error)` below.

Reply:
Isn't undefined the default anyway?

Reply:
Yes but JS will complain that the function requires two params if not provided.

Reply:
Actually, I am wrong. It'll be undefined.